Assets/SteamVR_Unity_Toolkit/Scripts/Abstractions/VRTK_WorldPointer.cs

```diff
@@ -182,7 +182,9 @@ protected virtual void PointerIn()
             OnDestinationMarkerEnter(SetDestinationMarkerEvent(pointerContactDistance, pointerContactTarget, destinationPosition, controllerIndex));
 
             interactableObject = pointerContactTarget.GetComponent<VRTK_InteractableObject>();
-            if (interactableObject && interactableObject.pointerActivatesUseAction && interactableObject.holdButtonToUse)
+            bool cannot_use_because_not_grabbed = (interactableObject.useOnlyIfGrabbed && !interactableObject.IsGrabbed());
```

Comment:
OK, have I really need to change cannot_use_because_not_grabbed to cannotUseBecauseNotGrabbed in this PR? Even if I seriously promise to change it in the next days with my next PR?... (please, have mercy...)